packages/common/driver-definitions/src/storage.ts

```diff
@@ -205,6 +206,14 @@ export interface IDocumentStorageService extends Partial<IDisposable> {
 	downloadSummary(handle: ISummaryHandle): Promise<ISummaryTree>;
 }
 
+/**
+ * Events emitted by {@link IDocumentService}.
+ * @alpha
+ */
+export interface IDocumentServiceEvents extends IEvent {
+	(event: "metadataUpdate", listener: (metadata?: Record<string, unknown> | undefined) => void);
```

Comment:
Some comment would be nice here. Maybe move description here, and have IContainer documentation refer to this event (not to duplicate documentation in two places).

Reply:
Added comment.